src/services/api/ProjectApiService.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosInstance, AxiosError } from 'axios';
+import { axios } from 'config/http';
+import { OperationStatus, ProjectDto } from './dtos';
+
+class ProjectApiService {
+  constructor(private readonly httpClient: AxiosInstance) {}
+
+  async getProjectById(id: string): Promise<OperationStatus<ProjectDto>> {
+    const token = localStorage.getItem('token')
+
+    try {
+      const response = await this.httpClient.get<ProjectDto>(
+        `/project/${id}`,
+        {
+          headers: { authorization: token },
+        }
```

Comment:
no need for this, axios instance has this header already set after login

Reply:
it seems it does not set it. maybe we should investigate if it works as you expect it.